materialize-elasticsearch/Dockerfile

```diff
@@ -11,6 +11,7 @@ RUN go mod download
 
 # Build the connector projects we depend on.
 COPY materialize-boilerplate ./materialize-boilerplate
+COPY go-schema-gen           ./go-schema-gen
```

Comment:
Probably too trivial a nit to be worth fixing, but maybe put `go-schema-gen` above `materialize-boilerplate` for alphabetical-ordering reasons (and likewise for the other Dockerfiles)?